sw/host/tests/manuf/manuf_scrap/src/main.rs

```diff
@@ -40,13 +40,18 @@ fn manuf_scrap(opts: &Opts, transport: &TransportWrapper) -> Result<()> {
         "Invalid initial LC state.",
     );
 
+    // CPU execution is disabled in the SCRAP state so we can safely reconnect
+    // to the LC TAP after the transition without risking the chip to be reset.
+    // FIXME it's unclear in the documentation if the LC TAP should work at all
```

Comment:
Maybe we can file/link an issue here to track?

Reply:
low priority issue of course

Reply:
I have created [an issue](#19550) for this. I will remove the comment for the code.